maintainers/scripts/dep-licenses.sh

```diff
@@ -38,10 +38,10 @@ if test $(wc -l "$tmp/filter.sed" | sed 's/ .*//') == 0; then
   echo 1>&2 "
 No derivation mentionned in the stack trace.  Either your derivation does
 not use stdenv.mkDerivation or you forgot to use the stdenv adapter named
-traceDrvLicenses.
+traceDrvLicenses. In pkgs/top-level/stdenv.nix:
 
--  defaultStdenv = allStdenvs.stdenv;
-+  defaultStdenv = traceDrvLicenses allStdenvs.stdenv;
+-  defaultStdenv = vanillaStdenv;
++  defaultStdenv = traceDrvLicenses avanillaStdenv;
 "
```

Comment:
Technically, I didn't even fix the error message to be a real diff. I was mainly interested in making sure my `allStdenvs` grep came clean before I removed it :).